game_state.go

```diff
@@ -92,6 +93,11 @@ func (gs GameState) IsWarmupPeriod() bool {
 	return gs.isWarmupPeriod
 }
 
+// IsGameStarted returns whether the match has started.
```

Comment:
```suggestion
// IsGameStarted returns whether the match has started according to CCSGameRulesProxy.
```

Never know if it's actually true with all those sketchy demos out there 😄 - to avoid questions in case it's preceived to be wrong.

Reply:
Haha, that's a good point!